controllers/controller.go

```diff
@@ -0,0 +1,36 @@
+package controller
+
+import (
+	"mytodoapp/database"
+	"mytodoapp/models"
+
+	"github.com/gin-gonic/gin"
+)
+
+func GetUsers(c *gin.Context) {
+	users := []models.User{}
+	database.DB.Find(&users)
+	c.JSON(200, &users)
```

Comment:
instead of returning this, lets have some standard payload that will be returned for all API responses